docs/installation.md

```diff
@@ -218,19 +218,174 @@ On Linux, as an optional post-installation task, you may wish to setup the bot t
 
 ------
 
-### Anaconda
+## Installation with Conda (Miniconda or Anaconda)
 
-Freqtrade can also be installed using Anaconda (or Miniconda).
+Freqtrade can also be installed with Miniconda or Anaconda. Conda (Miniconda or Anaconda) would automatically prepare and manage the extensive library-dependencies of the Freqtrade program.
```

Comment:
once the user is in this section, i'd assume this install-method is used - so "would" (hypothetical form) no longer applies, if you use this, then that's what it does ...


```suggestion
Freqtrade can also be installed with Miniconda or Anaconda. Conda will automatically prepare and manage the dependencies of Freqtrade.
```

Reply:
OK